recipes/perl-class-accessor/recipe.yaml

```
@@ -0,0 +1,48 @@
+context:
+  version: 0.51
+
+package:
+  name: perl-class-accessor
+  version: ${{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/K/KA/KASEI/Class-Accessor-${{ version }}.tar.gz
+  sha256: bf12a3e5de5a2c6e8a447b364f4f5a050bf74624c56e315022ae7992ff2f411c
+
+build:
+  number: 0
+  noarch: generic
+  script: |
+    perl Makefile.PL INSTALLDIRS=site NO_PERLLOCAL=1 NO_PACKLIST=1
+    make
+    make test
+    make install VERBINST=1
+
+requirements:
+  build:
+    - make
+  host:
+    - perl  # requires perl feedstock update for all versions
+  run:
+    - perl
+
+tests:
+  - perl:
+      uses:
+        - Class::Accessor
+        - Class::Accessor::Fast
+        - Class::Accessor::Faster
+
+about:
+  license: Artistic-1.0 OR GPL-1.0-or-later
+  license_file:
+    - ${{ environ["PREFIX"] }}/man/man1/perlartistic.1
+    - ${{ environ["PREFIX"] }}/man/man1/perlgpl.1
```

```suggestion
    - ${{ environ["PREFIX"] }}/man/man1/perlartistic.1
    - ${{ environ["PREFIX"] }}/man/man1/perlgpl.1
```
Unfortunately this doesn't seem to work with ratter recipes. For these `noarch` recipes, I think we don't need to worry about avoiding Unix commands because we only build them on Windows anyway.  And for the native recipes, I want to skip Windows for now because there are some issues with path handling of Perl packages on Windows anyway.